src/hangman.js

```diff
@@ -0,0 +1,24 @@
+export class Hangman {
+
+  constructor(rules, guessRegister) {
+    this.rules = rules
+    this.guessRegister = guessRegister
+  }
+
+  isOver() {
+    if (this.isGuessedWord() || this.areChancesToGuessOver()) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  isGuessedWord() {
+    return this.rules.getWord().split("").sort().join("") ==
+      this.guessRegister.correctGuesses.sort().join("")
+  }
+
+  areChancesToGuessOver() {
+    return this.guessRegister.wrongGuesses.length == 11;
```

Comment:
You may be aware but just in case, `==` doesn't have the same behaviour than other languages

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
Yeah, @felipesere told me about this just earlier!

Reply:
He is so fast!!!

Reply:
I think as a rule of thumb it's always better to use triple equals. Because it's more specific. So most people always use that unless you have a specific need for a double equals (which I have personally not come across)